SDL/Quintuplet.cu

```diff
@@ -1786,7 +1786,7 @@ __device__ bool SDL::runQuintupletDefaultAlgoBBBB(struct SDL::modules& modulesIn
 
     float alpha_OutUp,alpha_OutUp_highEdge,alpha_OutUp_lowEdge;
 
-    alpha_OutUp = SDL::deltaPhi(mdsInGPU.anchorX[fourthMDIndex], mdsInGPU.anchorY[fourthMDIndex], mdsInGPU.anchorX[fourthMDIndex] - mdsInGPU.anchorX[thirdMDIndex], mdsInGPU.anchorY[fourthMDIndex] - mdsInGPU.anchorY[thirdMDIndex]);
+    alpha_OutUp = SDL::phi_mpi_pi(atan2f(mdsInGPU.anchorY[fourthMDIndex] - mdsInGPU.anchorY[thirdMDIndex],mdsInGPU.anchorX[fourthMDIndex] - mdsInGPU.anchorX[thirdMDIndex])-mdsInGPU.anchorPhi[fourthMDIndex]);
```

Comment:
Maybe use the `phi` function here to understand better what's going on, instead of the `atan2`?

Only later I noticed that Gavin also had a similar comment below. I didn't really understand your arguments why not change it. If we are putting this variable in a function called `phi_mpi_pi`, we want to interepret that as φ and I believe it would be nice to make that clear.

I guess that part of the reason you don't want to change `atan2` to `phi` is because it is in so many places of the code. In this case, I don't have a strong opinion. However, if we decide that we want some kind of protection (as mentioned in the comment above), then a switch to `phi` would be necessary.

Reply:
For the idea of protection, I still think that we don't need it... We are giving the atan2 function an input of (x,y) and want it to return an angle between (-pi, pi), which is exactly the definition of the atan2 function in C++... So I'm a bit confused of why can't we simplify and what else do we need the phi to return..
For Gavin's suggestion of renaming all of them to "phi", I can change that to phi function and make it easy for people to understand.